pkg/agent/multicast/mcast_controller.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Antrea Authors
+// Copyright 2022 Antrea Authors
```

Comment:
The year in the copyright should be the year of first publication, please revert all changes.

Reply:
Do you mean all the copyright year changes should be reverted?

Reply:
Updated. All reverted.